src/Acuminator/Acuminator.Utilities/Roslyn/Semantic/Symbols/FieldAttributeSymbols.cs

```diff
@@ -10,6 +10,9 @@ public class FieldAttributeSymbols
 		/// </summary>
 		private const string PXDBPackedIntegerArrayAttributeFullName_Acumatica2018R2 = "PX.Data.PXDBPackedIntegerArrayAttribute";
 
+		private const string PeriodIDAttribute_Acumatica2019R1 = "PX.Objects.GL.PeriodIDAttribute";
```

Comment:
Why "Acumatica2019R1"? Isn't it the same in any other versions?
Also, if we move to string literals in all places, it doesn't make sense to extract them to separate constants - it decreases overall readability.